app/models.py

```diff
@@ -49,54 +50,75 @@
 If there are extra tools that you think project makers could benefit them, include them here.
 """
 
-
-class SocialLink(models.Model):
-    name = models.CharField(
-        max_length=24, verbose_name="Name", help_text="Name of this social link."
-    )
-    icon = models.CharField(
-        max_length=32, verbose_name="Icon", help_text="CSS FA icon class without 'fa-' prefix."
-    )
-    site = models.CharField(
-        max_length=2047,
-        verbose_name="Site",
-        help_text="Python format string which will have '%s' replaced with the link content.",
-    )
-    placeholder = models.CharField(
-        max_length=32,
-        verbose_name="Placeholder",
-        help_text="Placeholder text displayed to the user when creating a link.",
-    )
-
-    def __str__(self):
-        return self.name
+"""
+# FORMAT
+name: "Name of this social link"
+icon: "CSS FA icon class without 'fa-' prefix"
+regex: "regular expression to match this link"
+fab: True (optional; indicates whether the icon is part of FontAwesome Brand fonts; default is True)
+"""
+SOCIAL_LINKS = [
+    {"name": "Twitter", "icon": "twitter", "regex": r"(^(https?:\/\/)?)twitter\.com\/.*"},
+    {"name": "Facebook", "icon": "facebook", "regex": r"(^(https?:\/\/)?)facebook\.com\/.*"},
+    {"name": "Github", "icon": "github", "regex": r"(^(https?:\/\/)?)github\.com\/.*"},
+    {"name": "Reddit", "icon": "reddit", "regex": r"(^(https?:\/\/)?)reddit\.com\/.*"},
+    {"name": "Steam", "icon": "steam", "regex": r"(^(https?:\/\/)?)steamcommunity\.com\/.*"},
+    {"name": "YouTube", "icon": "youtube", "regex": r"(^(https?:\/\/)?)youtube\.com\/.*"},
+    {
+        "name": "Stack Overflow",
+        "icon": "stack-overflow",
+        "regex": r"(^(https?:\/\/)?)stackoverflow\.com\/.*",
+    },
+    {"name": "Vimeo", "icon": "vimeo-v", "regex": r"(^(https?:\/\/)?)vimeo\.com\/.*"},
+    {"name": "SoundCloud", "icon": "soundcloud", "regex": r"(^(https?:\/\/)?)soundcloud\.com\/.*"},
+    {"name": "Instagram", "icon": "instagram", "regex": r"(^(https?:\/\/)?)instagram\.com\/.*"},
+    {
+        "name": "Email",
+        "icon": "envelope",
+        "fab": False,
+        "regex": r"^(mailto:)?(.+@[a-zA-Z0-9\.-]+)$",
```

Comment:
Just a note that if you ignore case you can then remove the `A-Z` (or `a-z`) from here.

Reply:
`mailto:` should be mandatory here.